lib/atat-web-api-stack.ts

```diff
@@ -194,6 +196,18 @@ export class AtatWebApiStack extends cdk.Stack {
       );
     });
 
+    // Cloudwatch Logs for C5ISR
+    const logGroup = new logs.LogGroup(this, "LogGroup", {
+      logGroupName: "prod-cssp-cwl-logs",
+      retention: RetentionDays.INFINITE,
```

Comment:
Question: Do you need to set retention as infinite?
Based on the documentation, it seems that retention is set to infinite automatically: https://docs.aws.amazon.com/managedservices/latest/userguide/log-customize-retention.html

Reply:
You are right for console configuration but for cdk it has a 2 year default.  https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-logs.LogGroup.html and under 'retention?'